### ISSUE TITLE
Geometric Mean of Relative Overlap Frequencies

### ISSUE THREAD

**Issue by rogershijin** [mikessh/vdjtools#99]
I have a question regarding this page in the docs: https://vdjtools-doc.readthedocs.io/en/master/overlap.html
![image](https://user-images.githubusercontent.com/27650355/41180275-b0af31f6-6b3b-11e8-814e-0dc66dfab6e2.png)

![image](https://user-images.githubusercontent.com/27650355/41179468-caad53c4-6b38-11e8-9fd4-6b7e92f361b6.png)

![image](https://user-images.githubusercontent.com/27650355/41180247-9c58cf00-6b3b-11e8-8dc9-5efb8a72c9ab.png)

**Comment by mikessh** [comment]
You're absolutely right, ``k`` goes over indices of clonotypes that belong to both ``i`` and ``j``, so ``j`` is used implicitly. I'll replace it with your notation as it is more intuitive.
